src/plans/quasi_newton_plan.jl

```diff
@@ -413,6 +420,11 @@ function (d::QuasiNewtonMatrixDirectionUpdate{T})(
     get_vector!(M, r, p, -d.matrix \ get_coordinates(M, p, X, d.basis), d.basis)
     return r
 end
+function initialize_update!(d::QuasiNewtonMatrixDirectionUpdate)
```

Comment:
Would it be reasonable to put the matrix to copy over into the parameters here (positional, with default `I`)?

Reply:
That's somewhat unclear to me. It could probably be used for preconditioning but then maybe we would have to transport it between points upon reinitialization?

Reply:
Anyway, I'd prefer to do it in a separate PR.

Reply:
I am not sure what it could be used for, I just thought it might have been nice; but sure we can also keep that for a later PR, when we have a better idea what it could be used for.